python_modules/dagster/dagster/_core/host_representation/external_data.py

```diff
@@ -1345,6 +1380,29 @@ def external_repository_data_from_def(
     )
 
 
+def external_asset_checks_from_defs(
```

Comment:
Multiple JobDefinitions might have the same asset check spec, if a definition that includes it is targeted by multiple jobs. I think we'd want to make sure we don't end up with multiple `ExternalAssetCheck`s for that spec in that case/

Reply:
good call

Reply:
adding a test case